aws/resource_aws_kinesis_firehose_delivery_stream_test.go

```diff
@@ -1420,16 +1455,42 @@ resource "aws_kinesis_firehose_delivery_stream" "test_stream" {
 `, rInt, rInt, rInt, rInt, rInt, rInt)
 }
 
-var testAccKinesisFirehoseDeliveryStreamConfig_s3basic = testAccKinesisFirehoseDeliveryStreamBaseConfig + `
```

Comment:
Can you please create separate new test configurations instead of modifying existing ones? We should verify the existing configurations for regressions. Thanks.

Reply:
Done.